fares-data-build-tool/create-data
# feat: edit product name for period tickets

## Description

To allow users to edit period tickets' product names.

## Testing instructions

Run locally, create a period product, and then in My Fares attempt to edit it via the edit link. Point to point products should NOT be able to have their names edited.
